openhands/cli/commands.py

```diff
@@ -165,6 +171,8 @@ async def handle_commands(
         )
     elif command == '/mcp':
         await handle_mcp_command(config)
+    elif command == '/conv':
```

Comment:
```suggestion
    elif command == '/conversations':
```
It has autocomplete I think, so maybe it's nice to keep it human-friendly?